pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -659,6 +659,11 @@ private List<String> scheduleTask(PinotTaskGenerator taskGenerator, List<TableCo
       try {
         if (numTasks > 0) {
           // This might lead to lot of logs, maybe sum it up and move outside the loop
```

Comment:
comment should be before `LOGGER.info`

Reply:
this was for the log line after my patch. Reshuffled to avoid confusion.